manim/_config/utils.py

```diff
@@ -605,7 +605,7 @@ def digest_args(self, args: argparse.Namespace) -> "ManimConfig":
         if args.config_file:
             self.digest_file(args.config_file)
 
-        self.input_file = Path(args.file).absolute()
+        self.input_file = Path(args.file).absolute() if args.file != "-" else args.file
```

Comment:
I would rather not change this. Instead change 
https://github.com/ManimCommunity/manim/blob/14df62c3e1e210447642b911d4cba8b5c9686820/manim/utils/module_ops.py#L12
to 
```py
    if file_name.stem == "-":
```
But anyway the current one works.

Reply:
I think it doesn't make much sense for the input file to at any point be an absolute version of `-`, and the way this PR currently has it seems cleaner imo

Reply:
Agree with @friedkeenan. It would be weird to let `file_name` be one that doesn't even exist.